### PR TITLE
FEAT: verify & publish for Admin user

### PR DESCRIPTION
**Pull request by mazipan**
## What kind of change does this PR introduce?

Add flow to Verify and Publish -- for ADMIN only for now

## Changes

- [x] Show verified & publish status in the list
- [x] Add button to verify and publish

## Additional context

Screen Shot 2023-04-20 at 14 31 40
Screen Shot 2023-04-20 at 14 31 51

### CODE REVIEW

File: app/routes/admin.places._index.tsx [hunk 1]
```diff
@@ -41,10 +41,36 @@ export default function Route() {
...
+                className="card hover:card-hover block space-y-2"
               >
                 <h3>{place.name}</h3>
                 <code className="text-xs">{place.slug}</code>
+
+                <div className="queue-center text-xs">
+                  {place.isVerified ? (
+                    <Badge variant="brand" size="sm">
+                      <Check className="size-xs" />
+                      <span>Verified</span>
```
Comment (zainfathoni) [inline app/routes/admin.places._index.tsx:53]:
```suggestion
                      <span>Verified by {place.verifiedBy.user.name}</span>
```

Shall we also show the user name here?

**Review by zainfathoni — APPROVED** [review]
LGTM 💯 
But I'm wondering what it would look like to show the `verifiedBy` field after verifying the place.

File: app/routes/admin.places._index.tsx (hunk 1 above)
Comment (mazipan) [reply app/routes/admin.places._index.tsx:53]: Now the verifiedBy is not included when we select Place by Id.

So there is no detail data for that.

Comment (mazipan) [reply app/routes/admin.places._index.tsx:53]: Will exploring this later, since it needs to update the query as well.

Comment (zainfathoni) [reply app/routes/admin.places._index.tsx:53]: Thanks! For now, it's in an excellent state to be released as the first iteration of this feature.

**Pull request merged**